app/storage/metadata_parser.py

```diff
@@ -131,10 +129,6 @@ def convert_schema_name(self, data, **kwargs):
             logger.info(
                 "Using schema_name claim to specify schema, overriding survey, form_type and region_code"
             )
```

Comment:
Without the get_schema_name_from_census_params else can this function not be removed?

Reply:
Thanks, I was wondering how much census specific code was I supposed to remove as part of this PR..

Reply:
Removed now, thanks👍